SYCL/Plugin/level-zero-event-leak.cpp

```diff
@@ -0,0 +1,16 @@
+// REQUIRES: level_zero, level_zero_dev_kit
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %level_zero_options %s -o %t.out
+// RUN: env SYCL_DEVICE_FILTER=level_zero ZE_DEBUG=4 %GPU_RUN_PLACEHOLDER %t.out
+//
+// CHECK-NOT: ---> LEAK
+
+#include <CL/sycl.hpp>
+using namespace cl;
+int main(int argc, char **argv) {
+  sycl::queue Q;
+  const unsigned n_chunk = 129;
```

Comment:
Is 129 a magic number in any way?  I think that also deserves a comment.  Or maybe you could take that as a command line parameter, and run the executable multiple times with different values, and check for leakage with 3 or 4 different values?

Reply:
added comment